### PR TITLE
Improve workflow description in README

### PR DESCRIPTION
**Pull request by imobach**
Improved the workflow description in the README, as it was quite confusing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,72 +72,89 @@ Installation
...
 
-To run only one single test use:
+Generating a new ISO to use in tests is as easy as typing:
 
-    $ rake test[<path_to_test_file>]
+    $ rake build_iso[sles12-sp1]
 
-e.g. `rake test[test/sles12.rb]`
+where `sles12-sp1` could be any of the values defined in
+[config/definitions.yml](https://github.com/yast/autoyast-integration-test/blob/master/config/definitions.yml).
```
Comment (kobliha) [inline README.md]: I was checking whether we could use something like http://......./LATEST but unfortunately we can't as these are ONLY for unpacked media in SLP directory.

There are two problems:
1. config/definitions.yml needs to be maintained
2. dist.suse.de needs to be accessible, so maybe you could describe where to put an ISO if you already have it and how to name it?

This is just an idea, no real use-case as we (Yast team) actually HAVE access to dist.suse.de.

File: README.md [hunk 2]
```diff
@@ -76,72 +72,89 @@ Installation
...
+At this time SLE 12 (`sles12`) and SLE 12 SP1 (`sles12-sp1`) are supported.
 
-or you can also run any single script directly
+These are the steps that will be performed by this task:
 
-    $ bundle exec rspec [<path_to_test_file>]
+* If the original ISO is not found in the `iso` directory, it will be downloaded.
```
Comment (kobliha) [inline README.md]: See above, just PLS tell where exactly to put that ISO and how to name it.

File: README.md [hunk 3]
```diff
@@ -76,72 +72,89 @@ Installation
...
+* Latest YaST packages will also be downloaded from build system (IBS/OBS). If you
+  want to include your own packages, just drop them into `rpms/<definition>` directory
+  (e.g. `rpms/sles12-sp1`).
+* Those packages will be included in a DUD.
```
Comment (kobliha) [inline README.md]: This is the first mention about DUD, so better writer Driver Update (DUD) and reference that part in Linuxrc#p_dud wiki page.

File: README.md [hunk 4]
```diff
@@ -76,72 +72,89 @@ Installation
...
+  will take care of this part.
+* Using the generated Vagrant box, a new virtual machine will be created and
+  the tests will run on that machine.
+* The virtual machine will be destroyed.
 
-    # rmmod kvm
-    # rcvboxdrv start
+### Re-running the tests
+
+Sometimes is useful to run only a given test but skipping the installation
```
Comment (kobliha) [inline README.md]: Very good hint!

**Comment by kobliha** [comment]
I'm missing description for `rake clobber` as Jenkins run it, but this readme omits its description. Is it needed at all, do we need to run it?

**Comment by kobliha** [comment]
Anyway, even how it is now, LGTM, but if you enhance it a little bit more, the better :)
Thanks!

**Comment by imobach** [comment]
Thanks for your comments! I've updated the PR with that missing info.

**Comment by kobliha** [comment]
Nice! Please merge :) LGTM

**Pull request merged**